authentication/managing_cloud_provider_credentials/cco-mode-manual.adoc

```diff
@@ -6,23 +6,23 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-Manual mode is supported for Alibaba Cloud, Amazon Web Services (AWS), Microsoft Azure, Google Cloud Platform (GCP), IBM Cloud, and Nutanix.
+Manual mode is supported for Alibaba Cloud, Amazon Web Services (AWS), global Microsoft Azure, Microsoft Azure Stack Hub, Google Cloud Platform (GCP), IBM Cloud, and Nutanix.
 
 [id="manual-mode-classic_{context}"]
 == User-managed credentials
 
-In manual mode, a user manages cloud credentials instead of the Cloud Credential Operator (CCO). To use this mode, you must examine the `CredentialsRequest` CRs in the release image for the version of {product-title} that you are running or installing, create corresponding credentials in the underlying cloud provider, and create Kubernetes Secrets in the correct namespaces to satisfy all `CredentialsRequest` CRs for the cluster's cloud provider.
+In manual mode, a user manages cloud credentials instead of the Cloud Credential Operator (CCO). To use this mode, you must examine the `CredentialsRequest` CRs in the release image for the version of {product-title} that you are running or installing, create corresponding credentials in the underlying cloud provider, and create Kubernetes Secrets in the correct namespaces to satisfy all `CredentialsRequest` CRs for the cluster's cloud provider. Some platforms use the CCO utility (`ccoctl`) to ease this process during installation and updates.
 
-Using manual mode with long-term credentials allows each cluster component to have only the permissions it requires, without storing an administrator-level credential in the cluster. This mode also does not require connectivity to the AWS public IAM endpoint. However, you must manually reconcile permissions with new release images for every upgrade.
+Using manual mode with long-term credentials allows each cluster component to have only the permissions it requires, without storing an administrator-level credential in the cluster. This mode also does not require connectivity to services such as the AWS public IAM endpoint. However, you must manually reconcile permissions with new release images for every upgrade.
 
 For information about configuring your cloud provider to use manual mode, see the manual credentials management options for your cloud provider.
 
-[id="manual-mode-sts_{context}"]
-== Manual mode with cloud credentials created and managed outside of the cluster
-
-An AWS or GCP cluster that uses manual mode might be configured to create and manage cloud credentials from outside of the cluster using the AWS Security Token Service (STS) or GCP Workload Identity. With this configuration, the CCO uses short-term credentials for different components.
+[NOTE]
+====
+An AWS, global Azure, or GCP cluster that uses manual mode might be configured to create and manage cloud credentials from outside of the cluster. With this configuration, the CCO uses short-term credentials for different components.
```

Comment:
The wording here is a little confusing. I think it lost its meaning in the change. Of course, manual mode is always managed outside the cluster. I think what is important to note here is that these platforms might be configured to use short term credentials, as part of manual mode; and, when this is the case, these short term credentials will be used for different components.